.code-samples.meilisearch.yaml

```diff
@@ -3,6 +3,65 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+synonyms_guide_1: |-
+  let mut synonyms = std::collections::HashMap::new();
+  synonyms.insert(String::from("great"), vec![String::from("fantastic")]);
+  synonyms.insert(String::from("fantastic"), vec![String::from("great")]);
+
+  let task: TaskInfo = client
+    .index("movies")
+    .set_synonyms(&synonyms)
+    .await
+    .unwrap();
+date_guide_index_1: |-
+  let mut file = File::open("games.json")
+    .unwrap();
+  let mut content = String::new();
+  file
+    .read_to_string(&mut content)
+    .unwrap();
+  let docs: Vec<Game> = serde_json::from_str(&content)
```

Comment:
When you use `<Game>` this implies that you create a struct `Game` somewhere no?
Shouldn't we declare it for clarity?
```
  struct Game {
    ...
  }
```

Reply:
I was considering it, I wanted to avoid the code-sample to be to long. I also looked at the other samples using `Movies`, where the struct of `Movie` was not defined. 
For example in `.NET` 
https://github.com/meilisearch/meilisearch-dotnet/blob/79449225c454b335f8e550d82ba5f82eb2c033d3/.code-samples.meilisearch.yaml#L489